docs/src/getting-started/installation.md

```diff
@@ -441,3 +445,91 @@ the [source code](https://github.com/foundry-rs/starknet-foundry) as follows:
 2. Run `cd starknet-foundry && cargo build --release`. This will create a `target` directory.
 3. Move the `target` directory to the desired location (e.g. `~/.starknet-foundry`).
 4. Add `DESIRED_LOCATION/target/release/` to your `PATH`.
+
+# Uninstalling
+This section will guide you in how to uninstall Starknet Foundry.
+
+## Linux & MacOS
+If you installed **Starknet Foundry** using `asdf`, follow these steps to remove it.
```

Comment:
We don't provide any other installation instructions anyway. I'd just say something along the lines of "use these instructions to uninstall".